app/controllers/chat_channels_controller.rb

```diff
@@ -31,7 +31,11 @@ def create
 
```

Comment:
not sure this code is aligned with master though, if you look at 

https://github.com/thepracticaldev/dev.to/blob/master/app/controllers/chat_channels_controller.rb#L32-L35

the content of the method is different. Weird

Reply:
Hi @rhymes , this controller method is updated by @benhalpern , to allow mod users to update chat channel's description and than redirecting back to the edit view of chat channel membership. 

I have just added the code for checking if ChatChannel object is valid after updation, which previously used to be checked in `render_chat_channel` method